### PR TITLE
feat(chart): Add autoscaling resources

### PR DESCRIPTION
**Pull request by javirln**
This patch adds some missing autoscaling policies for `cas` and `controlplane`.

Diff:
```diff
diff --git a/old.yml b/new.yml
index d5669c1..e96fc7d 100644
--- a/old.yml
+++ b/new.yml
@@ -9,8 +9,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: postgresql
-    app.kubernetes.io/version: 16.3.0
-    helm.sh/chart: postgresql-15.5.20
+    app.kubernetes.io/version: 16.4.0
+    helm.sh/chart: postgresql-15.5.21
     app.kubernetes.io/component: primary
 spec:
   podSelector:
@@ -37,8 +37,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
-    app.kubernetes.io/version: 1.17.2
-    helm.sh/chart: vault-1.4.16
+    app.kubernetes.io/version: 1.17.3
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: injector
 spec:
@@ -67,8 +67,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
-    app.kubernetes.io/version: 1.17.2
-    helm.sh/chart: vault-1.4.16
+    app.kubernetes.io/version: 1.17.3
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: server
 spec:
@@ -123,8 +123,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: postgresql
-    app.kubernetes.io/version: 16.3.0
-    helm.sh/chart: postgresql-15.5.20
+    app.kubernetes.io/version: 16.4.0
+    helm.sh/chart: postgresql-15.5.21
     app.kubernetes.io/component: primary
 spec:
   maxUnavailable: 1
@@ -145,7 +145,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
     app.kubernetes.io/version: 1.4.2
-    helm.sh/chart: vault-1.4.16
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: injector
 spec:
@@ -167,8 +167,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
-    app.kubernetes.io/version: 1.17.2
-    helm.sh/chart: vault-1.4.16
+    app.kubernetes.io/version: 1.17.3
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: server
 spec:
@@ -205,8 +205,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: postgresql
-    app.kubernetes.io/version: 16.3.0
-    helm.sh/chart: postgresql-15.5.20
+    app.kubernetes.io/version: 16.4.0
+    helm.sh/chart: postgresql-15.5.21
 automountServiceAccountToken: false
 ---
 # Source: chainloop/charts/vault/templates/injector/service-account.yaml
@@ -220,7 +220,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
     app.kubernetes.io/version: 1.4.2
-    helm.sh/chart: vault-1.4.16
+    helm.sh/chart: vault-1.4.18
 automountServiceAccountToken: false
 ---
 # Source: chainloop/charts/vault/templates/server/service-account.yaml
@@ -233,8 +233,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
-    app.kubernetes.io/version: 1.17.2
-    helm.sh/chart: vault-1.4.16
+    app.kubernetes.io/version: 1.17.3
+    helm.sh/chart: vault-1.4.18
 automountServiceAccountToken: false
 ---
 # Source: chainloop/templates/cas/service-account.yaml
@@ -248,7 +248,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: chainloop
     app.kubernetes.io/version: v0.95.3
-    helm.sh/chart: chainloop-1.86.9
+    helm.sh/chart: chainloop-1.86.12
     app.kubernetes.io/component: cas
 ---
 # Source: chainloop/templates/controlplane/service-account.yaml
@@ -262,7 +262,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: chainloop
     app.kubernetes.io/version: v0.95.3
-    helm.sh/chart: chainloop-1.86.9
+    helm.sh/chart: chainloop-1.86.12
     app.kubernetes.io/component: controlplane
 ---
 # Source: chainloop/charts/dex/templates/secret.yaml
@@ -316,8 +316,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: postgresql
-    app.kubernetes.io/version: 16.3.0
-    helm.sh/chart: postgresql-15.5.20
+    app.kubernetes.io/version: 16.4.0
+    helm.sh/chart: postgresql-15.5.21
 type: Opaque
 data:
   password: "Y2hhaW5sb29wcHdk"
@@ -334,7 +334,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: chainloop
     app.kubernetes.io/version: v0.95.3
-    helm.sh/chart: chainloop-1.86.9
+    helm.sh/chart: chainloop-1.86.12
     app.kubernetes.io/component: cas
 type: Opaque
 stringData:
@@ -360,7 +360,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: chainloop
     app.kubernetes.io/version: v0.95.3
-    helm.sh/chart: chainloop-1.86.9
+    helm.sh/chart: chainloop-1.86.12
     app.kubernetes.io/component: cas
 type: Opaque
 data:
@@ -377,12 +377,12 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: chainloop
     app.kubernetes.io/version: v0.95.3
-    helm.sh/chart: chainloop-1.86.9
+    helm.sh/chart: chainloop-1.86.12
     app.kubernetes.io/component: controlplane
 type: Opaque
 data:
   # We store it also as a different key so it can be reused during upgrades by the common.secrets.passwords.manage helper
-  generated_jws_hmac_secret: "SHdPWXI1V2tJeA=="
+  generated_jws_hmac_secret: "NHhBcm5CRnBqUQ=="
   db_migrate_source: "cG9zdGdyZXM6Ly9jaGFpbmxvb3A6Y2hhaW5sb29wcHdkQGNoYWlubG9vcC1wb3N0Z3Jlc3FsOjU0MzIvY2hhaW5sb29wLWNwP3NzbG1vZGU9ZGlzYWJsZQ=="
 stringData:
   config.secret.yaml: |
@@ -406,7 +406,7 @@ stringData:
       # HMAC key used to sign the JWTs generated by the controlplane
       # The helper returns the base64 quoted value of the secret
       # We need to remove the quotes and then decoding it so it's compatible with the stringData stanza
-      generated_jws_hmac_secret: "HwOYr5WkIx"
+      generated_jws_hmac_secret: "4xArnBFpjQ"

       # Private key used to sign the JWTs meant to be consumed by the CAS
       cas_robot_account_private_key_path: "/secrets/cas.private.key"
@@ -421,7 +421,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: chainloop
     app.kubernetes.io/version: v0.95.3
-    helm.sh/chart: chainloop-1.86.9
+    helm.sh/chart: chainloop-1.86.12
     app.kubernetes.io/component: controlplane
 type: Opaque
 data:
@@ -437,8 +437,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
-    app.kubernetes.io/version: 1.17.2
-    helm.sh/chart: vault-1.4.16
+    app.kubernetes.io/version: 1.17.3
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: server
 data:
@@ -459,22 +459,22 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: chainloop
     app.kubernetes.io/version: v0.95.3
-    helm.sh/chart: chainloop-1.86.9
+    helm.sh/chart: chainloop-1.86.12
     app.kubernetes.io/component: cas
 data:
   server.yaml: |
     server:
       http:
-        addr: 0.0.0.0:8000
+        addr: "0.0.0.0:8000"
         # Timeouts for http downloads
         # grpc downloads/uploads don't require this because they don't have timeouts
         timeout: 300s
       grpc:
-        addr: 0.0.0.0:9000
+        addr: "0.0.0.0:9000"
         # Some unary RPCs are slow, so we need to increase the timeout
         timeout: 5s
       http_metrics:
-        addr: 0.0.0.0:5000
+        addr: "0.0.0.0:5000"
 ---
 # Source: chainloop/templates/controlplane/configmap.yaml
 apiVersion: v1
@@ -487,19 +487,19 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: chainloop
     app.kubernetes.io/version: v0.95.3
-    helm.sh/chart: chainloop-1.86.9
+    helm.sh/chart: chainloop-1.86.12
     app.kubernetes.io/component: controlplane
 data:
   config.yaml: |
     server:
       http:
-        addr: 0.0.0.0:8000
+        addr: "0.0.0.0:8000"
         timeout: 10s
         external_url: null
       http_metrics:
-        addr: 0.0.0.0:5000
+        addr: "0.0.0.0:5000"
       grpc:
-        addr: 0.0.0.0:9000
+        addr: "0.0.0.0:9000"
         timeout: 10s
     cas_server:
       grpc:
@@ -521,7 +521,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
     app.kubernetes.io/version: 1.4.2
-    helm.sh/chart: vault-1.4.16
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: injector
 rules:
@@ -543,7 +543,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
     app.kubernetes.io/version: 1.4.2
-    helm.sh/chart: vault-1.4.16
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: injector
 roleRef:
@@ -564,8 +564,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
-    app.kubernetes.io/version: 1.17.2
-    helm.sh/chart: vault-1.4.16
+    app.kubernetes.io/version: 1.17.3
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: server
 roleRef:
@@ -588,7 +588,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
     app.kubernetes.io/version: 1.4.2
-    helm.sh/chart: vault-1.4.16
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: injector
 rules:
@@ -641,8 +641,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
-    app.kubernetes.io/version: 1.17.2
-    helm.sh/chart: vault-1.4.16
+    app.kubernetes.io/version: 1.17.3
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: server
 rules:
@@ -695,8 +695,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
-    app.kubernetes.io/version: 1.17.2
-    helm.sh/chart: vault-1.4.16
+    app.kubernetes.io/version: 1.17.3
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: server
 rules:
@@ -715,7 +715,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
     app.kubernetes.io/version: 1.4.2
-    helm.sh/chart: vault-1.4.16
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: injector
 roleRef:
@@ -736,8 +736,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
-    app.kubernetes.io/version: 1.17.2
-    helm.sh/chart: vault-1.4.16
+    app.kubernetes.io/version: 1.17.3
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: server
   namespace: "platform-canary"
@@ -759,8 +759,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
-    app.kubernetes.io/version: 1.17.2
-    helm.sh/chart: vault-1.4.16
+    app.kubernetes.io/version: 1.17.3
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: server
   namespace: "platform-canary"
@@ -815,8 +815,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: postgresql
-    app.kubernetes.io/version: 16.3.0
-    helm.sh/chart: postgresql-15.5.20
+    app.kubernetes.io/version: 16.4.0
+    helm.sh/chart: postgresql-15.5.21
     app.kubernetes.io/component: primary
   annotations:
 spec:
@@ -845,8 +845,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: postgresql
-    app.kubernetes.io/version: 16.3.0
-    helm.sh/chart: postgresql-15.5.20
+    app.kubernetes.io/version: 16.4.0
+    helm.sh/chart: postgresql-15.5.21
     app.kubernetes.io/component: primary
 spec:
   type: ClusterIP
@@ -872,7 +872,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
     app.kubernetes.io/version: 1.4.2
-    helm.sh/chart: vault-1.4.16
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: injector
 spec:
@@ -900,8 +900,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
-    app.kubernetes.io/version: 1.17.2
-    helm.sh/chart: vault-1.4.16
+    app.kubernetes.io/version: 1.17.3
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: server
 spec:
@@ -935,8 +935,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
-    app.kubernetes.io/version: 1.17.2
-    helm.sh/chart: vault-1.4.16
+    app.kubernetes.io/version: 1.17.3
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: server
 spec:
@@ -966,8 +966,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
-    app.kubernetes.io/version: 1.17.2
-    helm.sh/chart: vault-1.4.16
+    app.kubernetes.io/version: 1.17.3
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: server
 spec:
@@ -1001,7 +1001,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: chainloop
     app.kubernetes.io/version: v0.95.3
-    helm.sh/chart: chainloop-1.86.9
+    helm.sh/chart: chainloop-1.86.12
     app.kubernetes.io/component: cas
   annotations:
     traefik.ingress.kubernetes.io/service.serversscheme: h2c
@@ -1030,7 +1030,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: chainloop
     app.kubernetes.io/version: v0.95.3
-    helm.sh/chart: chainloop-1.86.9
+    helm.sh/chart: chainloop-1.86.12
     app.kubernetes.io/component: cas
 spec:
   type: ClusterIP
@@ -1057,7 +1057,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: chainloop
     app.kubernetes.io/version: v0.95.3
-    helm.sh/chart: chainloop-1.86.9
+    helm.sh/chart: chainloop-1.86.12
     app.kubernetes.io/component: controlplane
   annotations:
     traefik.ingress.kubernetes.io/service.serversscheme: h2c
@@ -1086,7 +1086,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: chainloop
     app.kubernetes.io/version: v0.95.3
-    helm.sh/chart: chainloop-1.86.9
+    helm.sh/chart: chainloop-1.86.12
     app.kubernetes.io/component: controlplane
 spec:
   type: ClusterIP
@@ -1255,7 +1255,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
     app.kubernetes.io/version: 1.4.2
-    helm.sh/chart: vault-1.4.16
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: injector
 spec:
@@ -1275,7 +1275,7 @@ spec:
         app.kubernetes.io/managed-by: Helm
         app.kubernetes.io/name: vault
         app.kubernetes.io/version: 1.4.2
-        helm.sh/chart: vault-1.4.16
+        helm.sh/chart: vault-1.4.18
         app.kubernetes.io/part-of: vault
         app.kubernetes.io/component: injector
     spec:
@@ -1304,7 +1304,7 @@ spec:
         sysctls: []
       containers:
         - name: sidecar-injector
-          image: docker.io/bitnami/vault-k8s:1.4.2-debian-12-r4
+          image: docker.io/bitnami/vault-k8s:1.4.2-debian-12-r5
           imagePullPolicy: IfNotPresent
           securityContext:
             allowPrivilegeEscalation: false
@@ -1329,7 +1329,7 @@ spec:
             - name: AGENT_INJECT_VAULT_AUTH_PATH
               value: auth/kubernetes
             - name: AGENT_INJECT_VAULT_IMAGE
-              value: "docker.io/bitnami/vault:1.17.2-debian-12-r2"
+              value: "docker.io/bitnami/vault:1.17.3-debian-12-r0"
             - name: AGENT_INJECT_TLS_AUTO
               value: "chainloop-vault-platform-canary-injector"
             - name: AGENT_INJECT_TLS_AUTO_HOSTS
@@ -1383,7 +1383,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: chainloop
     app.kubernetes.io/version: v0.95.3
-    helm.sh/chart: chainloop-1.86.9
+    helm.sh/chart: chainloop-1.86.12
     app.kubernetes.io/component: cas
 spec:
   replicas: 2
@@ -1397,15 +1397,15 @@ spec:
   template:
     metadata:
       annotations:
-        checksum/config: b21da9d0d38ae90f582bb20086c9972b05b86f2bf18263ebd7206bdb5608ab6c
-        checksum/config-secret: 88bfb6fcdfcea52184b791b8957a1fe020836f50c8048a038965f6fe219f29de
-        checksum/public-key-secret: 58a6e2744c7ae69dadf2345013bbac0db40ea7ee25d182836350e28669ea7a53
+        checksum/config: 3d6bc37e25e5686367f28add83af192eae945a92f6130d5e4c47c564381189c4
+        checksum/config-secret: c104e3802f02795b82f0615b507862cd261f53b090240fc8a557dc46d180a38f
+        checksum/public-key-secret: cb67138bd42188c1d90e80edae8f52838d8d565646307847db21f563b784daa9
       labels:
         app.kubernetes.io/instance: chainloop
         app.kubernetes.io/managed-by: Helm
         app.kubernetes.io/name: chainloop
         app.kubernetes.io/version: v0.95.3
-        helm.sh/chart: chainloop-1.86.9
+        helm.sh/chart: chainloop-1.86.12
         app.kubernetes.io/component: cas
     spec:

@@ -1508,7 +1508,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: chainloop
     app.kubernetes.io/version: v0.95.3
-    helm.sh/chart: chainloop-1.86.9
+    helm.sh/chart: chainloop-1.86.12
     app.kubernetes.io/component: controlplane
 spec:
   replicas: 2
@@ -1522,16 +1522,16 @@ spec:
   template:
     metadata:
       annotations:
-        checksum/config: b78fd8ace3ab1f60e91fe81ca4fb9edd4cd92577109db572754cfd2976c838d6
-        checksum/secret-config: 948cf106f8686d3f2e22ac9b418a7d392510621a6e790c917c869a86760bc056
-        checksum/cas-private-key: 0e17d90e266498a5e23cf0a5403d7d7a807c6ddb534b4a2b0329d0b14bb89732
+        checksum/config: d89dc01fce2824edb62b7d05136b2f16f8a4b437a518321ce538d93127167140
+        checksum/secret-config: f5f56f518420c64cf689327213612500e8623242e1d64817b78102c5a9abe455
+        checksum/cas-private-key: 820cbc8a77c1507a5cb84b3ba959aa7edfecebb0e86b404bacb564a2e18c46e7
         kubectl.kubernetes.io/default-container: controlplane
       labels:
         app.kubernetes.io/instance: chainloop
         app.kubernetes.io/managed-by: Helm
         app.kubernetes.io/name: chainloop
         app.kubernetes.io/version: v0.95.3
-        helm.sh/chart: chainloop-1.86.9
+        helm.sh/chart: chainloop-1.86.12
         app.kubernetes.io/component: controlplane
     spec:

@@ -1656,8 +1656,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: postgresql
-    app.kubernetes.io/version: 16.3.0
-    helm.sh/chart: postgresql-15.5.20
+    app.kubernetes.io/version: 16.4.0
+    helm.sh/chart: postgresql-15.5.21
     app.kubernetes.io/component: primary
 spec:
   replicas: 1
@@ -1677,8 +1677,8 @@ spec:
         app.kubernetes.io/instance: chainloop
         app.kubernetes.io/managed-by: Helm
         app.kubernetes.io/name: postgresql
-        app.kubernetes.io/version: 16.3.0
-        helm.sh/chart: postgresql-15.5.20
+        app.kubernetes.io/version: 16.4.0
+        helm.sh/chart: postgresql-15.5.21
         app.kubernetes.io/component: primary
     spec:
       serviceAccountName: chainloop-postgresql
@@ -1708,7 +1708,7 @@ spec:
       hostIPC: false
       containers:
         - name: postgresql
-          image: docker.io/bitnami/postgresql:16.3.0-debian-12-r23
+          image: docker.io/bitnami/postgresql:16.4.0-debian-12-r0
           imagePullPolicy: "IfNotPresent"
           securityContext:
             allowPrivilegeEscalation: false
@@ -1841,8 +1841,8 @@ metadata:
     app.kubernetes.io/instance: chainloop
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
-    app.kubernetes.io/version: 1.17.2
-    helm.sh/chart: vault-1.4.16
+    app.kubernetes.io/version: 1.17.3
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: server
 spec:
@@ -1863,8 +1863,8 @@ spec:
         app.kubernetes.io/instance: chainloop
         app.kubernetes.io/managed-by: Helm
         app.kubernetes.io/name: vault
-        app.kubernetes.io/version: 1.17.2
-        helm.sh/chart: vault-1.4.16
+        app.kubernetes.io/version: 1.17.3
+        helm.sh/chart: vault-1.4.18
         app.kubernetes.io/part-of: vault
         app.kubernetes.io/component: server
     spec:
@@ -1894,7 +1894,7 @@ spec:
       initContainers:
       containers:
         - name: server
-          image: docker.io/bitnami/vault:1.17.2-debian-12-r2
+          image: docker.io/bitnami/vault:1.17.3-debian-12-r0
           imagePullPolicy: IfNotPresent
           securityContext:
             allowPrivilegeEscalation: false
@@ -2019,7 +2019,7 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: vault
     app.kubernetes.io/version: 1.4.2
-    helm.sh/chart: vault-1.4.16
+    helm.sh/chart: vault-1.4.18
     app.kubernetes.io/part-of: vault
     app.kubernetes.io/component: injector
 webhooks:
```

Refs #1151

### CODE REVIEW

File: deployment/chainloop/templates/cas/hpa.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{- /*
-Copyright Broadcom, Inc. All Rights Reserved.
```
Comment (migmartri) [inline deployment/chainloop/templates/cas/hpa.yaml:2]: ouch, thanks for the catch :)

File: deployment/chainloop/templates/cas/pdb.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "common.names.fullname" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  {{- if .Values.cas.pdb.minAvailable }}
```
Comment (migmartri) [inline deployment/chainloop/templates/cas/pdb.yaml:17]: do we have the `pdb` settings in the values.yaml file?

**Review by migmartri — APPROVED** [review]
LGTM, I'd just double check that we have all the required settings in the values.yaml file already, I am not sure if I added them when I migrated the deployment settings.

File: deployment/chainloop/templates/cas/pdb.yaml (hunk 2 above)
Comment (javirln) [reply deployment/chainloop/templates/cas/pdb.yaml:17]: yes we do!!

**Pull request merged**